pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -224,6 +224,17 @@ let
       UCLAMP_TASK_GROUP = yes;
     };
 
+    # All of these need to be enabled to support sched-ext
+    sched-ext = {
+      BPF = whenAtLeast "6.12" yes;
+      BPF_SYSCALL = yes;
+      SCHED_CLASS_EXT = whenAtLeast "6.12" yes;
+      BPF_JIT            = whenPlatformHasEBPFJit yes;
+      BPF_JIT_ALWAYS_ON  = whenPlatformHasEBPFJit (whenAtLeast "6.12" yes); # whenPlatformHasEBPFJit yes; # see https://github.com/NixOS/nixpkgs/issues/79304
```

Comment:
Why?

Reply:
It was disabled by default, due to `tests.installer.simpleUefiSystemdBoot.x86_64-linux` failing (https://github.com/NixOS/nixpkgs/issues/79304). I moved the option around and enabled it for 6.12+.

Reply:
I extremely doubt that it's actually necessary.

Reply:
Well the kernel docs recommend to enable it, that's why I made the change.

Reply:
I would like to see stronger justification than "the kernel docs say to enable it for this one feature". `CONFIG_SCHED_CLASS_EXT` does not actually depend on it, and enabling it comes with performance and security implications.